pkg/pillar/cmd/zedmanager/zedmanager.go

```diff
@@ -527,8 +527,6 @@ func handleCreate(ctxArg interface{}, key string,
 		ss := &status.StorageStatusList[i]
 		ss.UpdateFromStorageConfig(sc)
 		if ss.IsContainer {
-			// FIXME - We really need a top level flag to tell the app is
```

Comment:
Did it get less hacky somehow??

Reply:
I am no longer sure it is needed.. I guess I am more comfortable with the code now :-) My Earlier commits cleaned up a bunch of places where the Global flags were used. 

Hmm.. Thinking more, domain Mgr still needs to know the App is a container.. And there are other features coming - Like Environment Variable support - that are at the App Instance level ( Not just at storage level ) - So may be it is too early to remove this comment.

Let me remove this change.. As part of the Environment variable support, we will add that global flag..